web/src/labs/marked/parser/Blockquote.ts

```diff
@@ -1,6 +1,6 @@
 import { escape } from "lodash";
 
-export const BLOCKQUOTE_REG = /^>\s+([\S ]+)(\n?)/;
+export const BLOCKQUOTE_REG = /^>\s+([\S \u3000]+)(\n?)/;
```

Comment:
I think a better way is that we should replace all of the full width space `"　"` with normal space before rendering memo content. So that we don't need to handle a special character in parser level.

Reply:
This might be a way, but if we consider this issue #425 , I think he wants to use full-width spaces to align his text (Chinese characters).

So maybe not the best for him if we replace them with normal spaces.

Reply:
Then we could use two half width space to replace the special full width space. WDYT

Reply:
In fact, in Chinese typography, the width of two half-width spaces is not equal to the width of one full-width space, because there in space between two characters. 

Like this 
![image](https://user-images.githubusercontent.com/31177490/200982369-784c8136-754b-4732-b76d-1c6cb71220be.png)

Maybe it's a small problem, but not perfect I think.